### PR TITLE
Juxebox playtime

### PR DESCRIPTION
**Pull request by zeno4ever**

### CODE REVIEW

**Comment by zeno4ever** [comment]
Another try see if you can use it .. You already warned me that you are trowing things around in the code ;-)

File: README.md
Comment (creativenucleus) [inline README.md:1]: Thanks. Appreciate the commenting I didn't yet get round to =)

File: jukebox.go [hunk 1]
```diff
@@ -62,6 +64,22 @@ func (j *Jukebox) start() {
 					fmt.Printf("Description: %s\n", playlistItem.description)
 				}
 
+				var timeoutduration time.Duration
+				//from json file
+				if playlistItem.playtime > 0 {
+					timeoutduration = time.Duration(playlistItem.playtime) * time.Second
+				}
+				//from command prompt
```
Comment (creativenucleus) [inline jukebox.go:72]: I jigged this around - the Jukebox has a default playtime, and this is now set from a constant, so it's only the playlistItem.playtime that needs to be tested as a special case

File: main.go [hunk 2]
```diff
@@ -57,15 +58,22 @@ func main() {
 						Name:  "playlist",
 						Usage: "Playlist file (empty for LCDZ playlist)",
 					},
+					&cli.StringFlag{
```
Comment (creativenucleus) [inline main.go:61]: I switched these into uint flags, and popped a constant at the top of the file that is used as default values - the CLI package is pretty good at self documenting, so I've let that do its job :)

**Pull request merged**